decks/ansible-essentials.html

```diff
@@ -423,7 +423,7 @@ <h2>Static Inventory Example</h2>
           <h2>Static Inventory Example</h2>
           <pre class="language-yaml"><code>
 [control]
-control ansible_host=10.42.0.2
+tower ansible_host=10.42.0.2
```

Comment:
yes that makes total sense, I think this was a mistake and leftover because we call it ansible everywhere else.  Fixed in commit: https://github.com/network-automation/lightbulb/commit/add0aacfcdfb382bf2aa599e6e9ad0c800cabaa6